tests/system/small/test_series.py

```diff
@@ -2766,22 +2766,17 @@ def test_series_case_when(scalars_dfs_maybe_ordered):
     bf_series = scalars_df["int64_col"]
     pd_series = scalars_pandas_df["int64_col"]
 
-    # TODO(tswast): pandas case_when appears to assume True when a value is
```

Comment:
I would keep this comment

Reply:
Updated to have all cases in the original test. Keep only one test can be overall faster I think.